English-language wiki documentation, revisions for grammar and clarity (PR Principles).

hawthord opened the pull request:
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 👻 What's the background?

1. The English-language wiki documentation for the PR Principles [https://github.com/ant-design/ant-design/wiki/PR-principle](https://github.com/ant-design/ant-design/wiki/PR-principle) is somewhat unclear and may make it difficult for first-time contributors to make a pull request.

### 💡 Solution

1. I suggest revising the wiki text as per the attached HTML document, lines 835-885.
[PR principle · ant-design_ant-design Wiki.zip](https://github.com/ant-design/ant-design/files/3205201/PR.principle.ant-design_ant-design.Wiki.zip)

### 📝 Changelog

The user side changes consist of a revision to the English-language PR principles documented on the Ant Design wiki at https://github.com/ant-design/ant-design/wiki/PR-principle. There are no risks of website breakage, as all changes are simple HTML body text modifications.

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |x|
| 🇨🇳 Chinese | |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed


-----
[View rendered docs/spec/values.en-US.md](https://github.com/ant-design/ant-design/blob/master/docs/spec/values.en-US.md)

christophehurpeau committed:
Upload prop onRemove: allow Promise as return type

renovate-bot committed:
Add renovate.json

afc163 committed:
Merge pull request #16675 from ant-design/renovate/configure

Configure Renovate

afc163 committed:
Update renovate.json

gotjoshua committed:
Refined translation of St.Exupery quote 

including link to quote site

ycjcl868 committed:
fix horizontal divider style

afc163 committed:
Merge pull request #16681 from gotjoshua/patch-1

Refined translation of St.Exupery quote

afc163 committed:
Merge pull request #16684 from ant-design/fix-divider-style

fix horizontal divider style

yoyo837 committed:
fix shouldComponentUpdate

afc163 committed:
Merge pull request #16712 from yoyo837/fix-16706

fix shouldComponentUpdate

christophehurpeau committed:
Upload prop onRemove: fix action

https://github.com/react-component/upload/tree/d0cf0eecee3b31888791f8c57acddb575bd259d4#api

afc163 committed:
Merge pull request #16570 from christophehurpeau/patch-1

fix: Upload prop onRemove: allow Promise as return type

afc163 committed:
Merge pull request #16716 from christophehurpeau/patch-2

fix: Upload prop action: fix typings

renovate-bot committed:
Update dependency logrocket to v1

afc163 committed:
Merge pull request #16718 from ant-design/renovate/logrocket-1.x

Update dependency logrocket to v1

afc163 committed:
:lipstick: chore some documents style

afc163 committed:
:memo: update customize theme

afc163 reviewed (changes requested):
Mixed other commits.

afc163 committed:
Merge pull request #16727 from ant-design/chore-doc

chore: some documents style

afc163 committed:
:clapper: Add example for Select ``optionLabelProp` prop, #16709

afc163 committed:
:white_check_mark: update snapshots

afc163 committed:
:lipstick: fix emoji lint problem

https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/accessible-emoji.md

afc163 committed:
Merge pull request #16737 from ant-design/add-select-optionLabelProp-demo

docs: Add select optionLabelProp demo

zombieJ committed:
adjust Carousel children change goto Logic (#16756)

afc163 committed:
:memo: Add Programmatic Usage of antd colors

afc163 committed:
Merge pull request #16775 from ant-design/doc-antd-colors

docs: 📝 Add Programmatic Usage of antd colors

zombieJ committed:
update preview demo

afc163 committed:
Merge pull request #16777 from ant-design/fix-upload-demo

docs: update preview demo

afc163 committed:
:bug: Fix Cascader displayRender not interactive

close #16738 and #10433

this bug had been fixed in bc4143f before, but it was broken by #12407

here we change another method to fix #12395

afc163 committed:
Merge pull request #16782 from ant-design/fix-cascader-custom-item

fix: 🐛 Fix Cascader displayRender not interactive

afc163 committed:
Create FUNDING.yml

afc163 committed:
:memo: Add opencollective link

Veveue committed:
upload disabled remove

afc163 committed:
Merge pull request #16786 from Veveue/master

fix: upload disabled remove

zombieJ commented:
Close since it seems only merge `master` into `feature`.
